demo/src/main/java/com/novoda/storagepathfinder/demo/DemoDependenciesFactory.java

```diff
@@ -0,0 +1,40 @@
+package com.novoda.storagepathfinder.demo;
+
+import android.content.Context;
+import android.content.res.AssetManager;
+
+import com.novoda.storagepathfinder.AndroidApplicationDirectories;
+import com.novoda.storagepathfinder.AndroidDeviceFeatures;
+import com.novoda.storagepathfinder.AndroidDeviceStorageInspector;
+import com.novoda.storagepathfinder.AndroidFileSystem;
+import com.novoda.storagepathfinder.AndroidSystem;
+import com.novoda.storagepathfinder.CommonDirectories;
+import com.novoda.storagepathfinder.DeviceFeatures;
+import com.novoda.storagepathfinder.FileSystem;
+
+import java.util.concurrent.Executors;
+
+public final class DemoDependenciesFactory {
```

Comment:
Are you sure about this? This class would go balloon if we add more dependencies...

Reply:
It's for the demo. If it balloons then we should add another dependency management system 😄